x-pack/packages/security-solution/ecs_data_quality_dashboard/tsconfig.json

```diff
@@ -22,5 +22,7 @@
     "@kbn/ui-theme",
     "@kbn/core-http-browser",
     "@kbn/core-http-browser-mocks",
+    "@kbn/elastic-assistant",
+    "@kbn/triggers-actions-ui-plugin",
```

Comment:
Why do we need the reference for `triggers-actions-ui-plugin` here?

Reply:
IIRC it was needed because of the mock for tests.